### PR TITLE
Update to Chromium 128.0.6613.137

### PR DESCRIPTION
**Pull request by Ahrotahn**
This PR updates to Chromium 128.0.6613.137
This also includes a small fix to the RemoveClientHints patch which returns an empty object instead of null for userAgentData since some extensions don't expect that to be null.  

<br>

[Change log](https://chromium.googlesource.com/chromium/src/+log/128.0.6613.119..128.0.6613.137?pretty=fuller&n=10000)
[Blog post](https://chromereleases.googleblog.com/2024/09/stable-channel-update-for-desktop_10.html)

### CODE REVIEW

**Comment by clickot** [comment]
portable linux builds fine and runs at least on Manjaro, Ubuntu 20.04/24.04 and Fedora WS 40

**Comment by networkException** [comment]
can do

File: patches/extra/ungoogled-chromium/add-flag-to-remove-client-hints.patch [hunk 1]
```diff
@@ -59,12 +59,12 @@
...
- namespace blink {
- 
+@@ -14,6 +15,8 @@ namespace blink {
  NavigatorUAData* NavigatorUA::userAgentData() {
-+  if (base::FeatureList::IsEnabled(blink::features::kRemoveClientHints))
-+    return nullptr;
    NavigatorUAData* ua_data =
        MakeGarbageCollected<NavigatorUAData>(GetUAExecutionContext());
++  if (base::FeatureList::IsEnabled(blink::features::kRemoveClientHints))
++    return ua_data;
```
Comment (networkException) [inline patches/extra/ungoogled-chromium/add-flag-to-remove-client-hints.patch:67]: This condition looks inverted in reference to the previous iteration of the patch

Comment (Ahrotahn) [reply patches/extra/ungoogled-chromium/add-flag-to-remove-client-hints.patch:67]: ua_data is a newly initialized object which gets filled in below that section, so everything still returns empty:
Screenshot

**Review by PF4Public — APPROVED** [review]

**Review by networkException — APPROVED** [review]

**Pull request merged**